nsxt/resource_nsxt_policy_ldap_identity_source_test.go

```diff
@@ -0,0 +1,226 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	nsxModel "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+var accTestPolicyLdapIdentitySourceCreateAttributes = map[string]string{
+	"display_name": getAccTestResourceName(),
+	"description":  "terraform created",
+}
+
+var accTestPolicyLdapIdentitySourceUpdateAttributes = map[string]string{
+	"display_name": getAccTestResourceName(),
+	"description":  "terraform updated",
+}
+
+func TestAccResourceNsxtPolicyLdapIdentitySource_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_ldap_identity_source.test"
+	ldapType := nsxModel.LdapIdentitySource_RESOURCE_TYPE_ACTIVEDIRECTORYIDENTITYSOURCE
```

Comment:
Can we test both types?

Reply:
Do you think we make 2 sets of ENVs (one set for open LDAP another for AD)? Or just add another ENV var to select which type of server to test against?

Reply:
Would the same set of ENV variable work for both types? In this case, I would suggest to reuse same vars for both types

Reply:
`bind_identity` will be different. It all boils down to how we expect this acc test to be run in our pipelines in the future. If we want to cover both in one go then we need 2 servers (openldap + ad) anyways, so 2 sets of VARs. Otherwise the VAR names makes sense for both server types. Just different values should be set to them.